cli.js

```diff
@@ -44,6 +44,10 @@ const cli = meow(help, {
       type: 'boolean',
       alias: 's'
     },
+    clip: {
+      type: 'boolean',
+      alias: 'g'
+    },
```

Comment:
To make things, as clear and obvious as possible, for the readers, we could go with `copy` for the main option name and `y` for alias, mainly for its resemblance to vim's widely use `yank` command.

```js
copy: {
  type: boolean,
  alias: 'y'
},